src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/BeneficialOwnerCorporateValidatorTest.java

```diff
@@ -251,11 +259,57 @@ void testErrorReportedWhenNoNatureOfControlValuesAreAllNull() {
+        beneficialOwnerCorporateDtoList.get(0).setNonLegalFirmMembersNatureOfControlTypes(null);
+        beneficialOwnerCorporateDtoList.get(0).setTrusteesNatureOfControlTypes(null);
+        beneficialOwnerCorporateDtoList.get(0).setNonLegalFirmControlNatureOfControlTypes(null);
+        beneficialOwnerCorporateDtoList.get(0).setTrustControlNatureOfControlTypes(null);
+        beneficialOwnerCorporateDtoList.get(0).setOwnerOfLandPersonNatureOfControlJurisdictions(null);
+        beneficialOwnerCorporateDtoList.get(0).setOwnerOfLandOtherEntityNatureOfControlJurisdictions(null);
+        Errors errors = beneficialOwnerCorporateValidator.validate(beneficialOwnerCorporateDtoList, new Errors(), LOGGING_CONTEXT);
+        String qualifiedFieldName = getQualifiedFieldName(
+                BENEFICIAL_OWNERS_CORPORATE_FIELD,
+                BeneficialOwnerCorporateValidator.NATURE_OF_CONTROL_FIELDS);
+        String validationMessage = String.format(ValidationMessages.NOT_EMPTY_ERROR_MESSAGE, qualifiedFieldName);
+
+        assertError(BeneficialOwnerCorporateValidator.NATURE_OF_CONTROL_FIELDS, validationMessage, errors);
+    }
+
+    // Changed for UAR-1597 can remove test when FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC is removed
+    @Test
+    void testErrorReportedWhenNoNatureOfControlValuesAreAllEmptyWhenNewNocsFlagIsFalse() {
+        setNewNocsEnabledFeatureFlag(false);
+        beneficialOwnerCorporateDtoList.get(0).setBeneficialOwnerNatureOfControlTypes(new ArrayList<>());
+        beneficialOwnerCorporateDtoList.get(0).setNonLegalFirmMembersNatureOfControlTypes(new ArrayList<>());
+        beneficialOwnerCorporateDtoList.get(0).setTrusteesNatureOfControlTypes(new ArrayList<>());
+        beneficialOwnerCorporateDtoList.get(0).setNonLegalFirmControlNatureOfControlTypes(new ArrayList<>());
+        beneficialOwnerCorporateDtoList.get(0).setTrustControlNatureOfControlTypes(new ArrayList<>());
+        beneficialOwnerCorporateDtoList.get(0).setOwnerOfLandPersonNatureOfControlJurisdictions(new ArrayList<>());
+        beneficialOwnerCorporateDtoList.get(0).setOwnerOfLandOtherEntityNatureOfControlJurisdictions(new ArrayList<>());
+        Errors errors = beneficialOwnerCorporateValidator.validate(beneficialOwnerCorporateDtoList, new Errors(), LOGGING_CONTEXT);
+        String qualifiedFieldName = getQualifiedFieldName(
+                BENEFICIAL_OWNERS_CORPORATE_FIELD,
+                BeneficialOwnerCorporateValidator.NATURE_OF_CONTROL_FIELDS);
+        String validationMessage = String.format(ValidationMessages.NOT_EMPTY_ERROR_MESSAGE, qualifiedFieldName);
+
+        assertError(BeneficialOwnerCorporateValidator.NATURE_OF_CONTROL_FIELDS, validationMessage, errors);
+    }
+
+    // Added for UAR-1597 can remove test when FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC is removed
+    @Test
+    void testErrorReportedWhenNoNatureOfControlValuesAreAllEmptyWhenNewNocsFlagIsTrue() {
```

Comment:
similar for the 'No' in the title on this one

Reply:
Removed all instances including those in the GPA tests.